lodash.js

```diff
@@ -10424,6 +10424,41 @@
     }
 
     /**
+     * Converts a `value` to an array.
+     *
+     * @static
+     * @memberOf _
+     * @category Lang
+     * @param {*} value The value to convert.
+     * @returns {Array} Returns the converted array.
+     * @example
+     *
+     * _.toArray({ 'a': 1, 'b': 2 });
+     * // => [{ 'a': 1, 'b': 2 }]
+     *
+     * _.toArray([1, 2, 3]);
+     * // => [1, 2, 3]
+     *
+     * _.toArray('abc');
+     * // => ['abc']
+     *
+     * _.toArray(1);
+     * // => [1]
+     *
+     * _.toArray(null);
+     * // => [null]
+     */
+    function asArray(value) {
+      if (isArray(value)) {
+        return copyArray(value);
+      }
+      if (iteratorSymbol && value && value[iteratorSymbol] && !isString(value)) {
+        return iteratorToArray(value[iteratorSymbol]());
+      }
+      return [value];
+    }
+
+    /**
```

Comment:
Cool. Adding shallow cloning is easy enough before we bump. I'll ask for feedback on the tweets.